pages/src/landing/marquee/marquee.ts

```diff
@@ -0,0 +1,63 @@
+import {ESLBaseElement, attr} from '../../../../src/modules/esl-base-element/core';
+import {bind} from '../../../../src/modules/esl-utils/decorators/bind';
+import {ready} from '../../../../src/modules/esl-utils/decorators/ready';
+import {range} from '../../../../src/modules/esl-utils/misc/array';
+import {memoize} from '../../../../src/modules/esl-utils/decorators/memoize';
+import {isIE} from '../../../../src/modules/esl-utils/environment/device-detector';
+
+export class ESLDemoMarquee extends ESLBaseElement {
+  static is = 'esl-d-marquee';
+  static STARS_SEL = [
+    '#esl-logo-shield-stars > path',
+    '#esl-logo-border-inner-stars > path',
+    '#esl-logo-border-outer-stars > path',
+    '#esl-logo-wrench-left-stars > path',
+    '#esl-logo-wrench-right-stars > path'
+  ].join(',');
+
+  @attr({defaultValue: '4'}) public targetsNumber: string;
+  @attr({defaultValue: '3000'}) public iterationTime: string;
+
+  private _$active: HTMLElement[] = [];
+  private _animateTimer: number = 0;
+
+  @ready
+  protected connectedCallback() {
+    super.connectedCallback();
+    if (isIE) return;
+    this.startAnimation();
+  }
+  protected disconnectedCallback() {
+    this.stopAnimation();
```

Comment:
It seems that under IE we don't need to stop the animation, because we didn't start it.

```suggestion
    if (!isIE) this.stopAnimation();
```

Reply:
`stopAnimation `is safe function, in case animation wasn't started if will do nothing.